stable/database/files/nuosm

```diff
@@ -12,8 +12,6 @@
 : ${DB_NAME:="demo"}
 : ${NUODB_SEQUENCE_SYNC:=true}
 
-: ${NUODB_IMPORT_CREDENTIALS:=:}
```

Comment:
GitHub diff viewer is not doing a good job with this file changes. You might want to review this file using "split" more which is side-by-side is a bit easier.
I will also put markers on which part should be reviewed.